src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts

```diff
@@ -73,7 +73,9 @@ export const validateDataCompletion = (req: AppRequest<Partial<Case>>): void =>
     if (fieldsArray.includes(key)) {
       const value = req.session.userCase[`${key}`];
       if (typeof value === 'string' && (value === null || value === undefined || value.trim() === '')) {
-        req.session.userCase[`${key}`] = '<span class="govuk-error-message">Complete this section</span>';
+        req.session.lang === 'cy'
+          ? (req.session.userCase[`${key}`] = '<span class="govuk-error-message">Complete this section (welsh)</span>')
```

Comment:
Can you please get "Complete this section" from content.ts and not directly hardcode?

Reply:
Changed to use content.ts